src/components/pipeline/CreatePipeline.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+
+import Main from '../common/Main';
+import BackButton from '../common/BackButton';
+import { EDIT_PIPELINE_BACK_BUTTON_ID } from '../../config/selectors';
+
+import PipelineForm from './PipelineForm';
+
+const styles = (theme) => ({
+  backButton: {
+    position: 'fixed',
+    right: theme.spacing(2),
+    bottom: theme.spacing(2),
+  },
+  content: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    flexDirection: 'column',
+  },
+});
+
+const CreatePipeline = (props) => {
+  const { t, classes } = props;
+
+  return (
+    <Main>
+      <Container className={classes.content}>
+        <Typography variant="h4">{t('Create Pipeline')}</Typography>
+        <PipelineForm pipelineAlgorithms={[]} name="" description="" />
+      </Container>
+      <BackButton
+        className={classes.backButton}
+        id={EDIT_PIPELINE_BACK_BUTTON_ID}
```

Comment:
Either use `CREATE_PIPELINE_BACK_BUTTON_ID` or use a general one for both pages `PIPELINE_BACK_BUTTON_ID`